### PR TITLE
Feature/KITOSUDV-3635

### PR DESCRIPTION
**Pull request by a-naskret**
## KITOS Pull request template
The following procedure dictates the steps needed before a Pull request can be merged into master.

- [x] **Implement**: 
      *All requirements are implemented and unit tests are green*
      
- [x] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [x] **Green on integration**: 
      *All integration tests are green on integration*
      
- [x] **Request review**: 
      *Tag whomever you wish to review your code*
      
- [ ] **Review completed**: 
      *Reviewer ticks this box when review comments have been submitted*
      
- [ ] **Changes**: 
      *PR owner and reviewer agrees on which changes must be made and the changes are committed.*
      
- [ ] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [ ] **Green on integration**: 
      *All integration tests are green on integration*

### CODE REVIEW

File: Presentation.Web/app/components/org/user/org-user.controller.ts [hunk 1]
```diff
@@ -376,6 +396,7 @@
                             "data-element-type": "rightsHolderHeader"
                         },
                         template: (dataItem) => setBooleanValue(dataItem.isRightsHolder),
+                        excelTemplate: (dataItem) => dataItem.isRightsHolder ? "Ja" : "Nej",
```
Comment (mrjsawdk) [inline Presentation.Web/app/components/org/user/org-user.controller.ts:399]: duplication.... remember DRY DRY

File: Presentation.Web/app/components/org/user/org-user.controller.ts [hunk 2]
```diff
@@ -419,6 +446,13 @@
             this.mainGridOptions = mainGridOptions;
         }
 
+        private readonly excelConfig: Models.IExcelConfig = {
```
Comment (mrjsawdk) [inline Presentation.Web/app/components/org/user/org-user.controller.ts:449]: is this used for anything other than being passed as an object... my point is.. does it have to be an object which lives on the class or could it just be a `{}` passed to the call?

File: Presentation.Web/app/components/org/user/org-user.controller.ts [hunk 3]
```diff
@@ -307,6 +312,17 @@
                             }
                             return `<span data-ng-model="dataItem.OrganizationUnitRights" value="rights.Role.Name" ng-repeat="rights in dataItem.OrganizationUnitRights"> {{rights.Role.Name}}{{$last ? '' : ', '}}</span>`;
                         },
+                        excelTemplate: (dataItem) => {
+                            var rightNames = "";
+                            dataItem.OrganizationUnitRights.forEach((right, index) => {
+                                rightNames += right.Role.Name;
```
Comment (mrjsawdk) [inline Presentation.Web/app/components/org/user/org-user.controller.ts:318]: why not use join in stead?
`dataItem.OrganizationUnitRights.map(right=>right.Role.Name).join(", ");`

That takes care of the "last item" issue

**Review by mrjsawdk — CHANGES_REQUESTED** [review]
@a-naskret some quick feedback before I test

File: Presentation.Web/app/components/org/user/org-user.controller.ts [hunk 4]
```diff
@@ -418,6 +435,10 @@
 
             this.mainGridOptions = mainGridOptions;
         }
+        
+        private exportToExcel = (e: IKendoGridExcelExportEvent<Models.ItSystem.IItSystem>) => {
```
Comment (mrjsawdk) [inline Presentation.Web/app/components/org/user/org-user.controller.ts:439]: "Kun viste kolonner" does not work... so something is offf in this one.. Works fine in IT-System catalog using the same integration... 

image

**Review by mrjsawdk — CHANGES_REQUESTED** [review]

File: Presentation.Web/app/components/org/user/org-user.controller.ts (hunk 2 above)
Comment (a-naskret) [reply Presentation.Web/app/components/org/user/org-user.controller.ts:439]: this one is used to store the visibility config

**Pull request merged**